wrappers/ios/Sources/Wrapper/ZXIFormat.h

```diff
@@ -0,0 +1,15 @@
+// Copyright 2022 KURZ Digital Solutions GmbH
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#import <Foundation/Foundation.h>
+#ifndef ZXIFormat_h
+#define ZXIFormat_h
+
+typedef NS_ENUM(NSInteger, ZXIFormat) {
+    NONE, AZTEC, CODABAR, CODE_39, CODE_93, CODE_128, DATA_BAR, DATA_BAR_EXPANDED,
+    DATA_MATRIX, EAN_8, EAN_13, ITF, MAXICODE, PDF_417, QR_CODE, MICRO_QR_CODE, UPC_A, UPC_E,
+    ONE_D_CODES, TWO_D_CODES, ANY
```

Comment:
I just recently replaced the terms "OneD"/"TwoD" with "Linear" and "Matrix" as they are used in the standard specs. Please follow that change. See also the deprecation warning you currently get.